promptsource/promptsource.py

```diff
@@ -143,22 +143,13 @@ def show_text(t, width=WIDTH):
             state.working_priority_ds = None
             priority_max_templates = None
 
-    dataset_list = list_datasets(
-        template_collection,
-        priority_filter,
-        priority_max_templates,
-        state,
-    )
+    dataset_list = list_datasets(template_collection, priority_filter, priority_max_templates, state,)
 
     #
     # Select a dataset - starts with ag_news
     #
     dataset_key = st.sidebar.selectbox(
-        "Dataset",
-        dataset_list,
-        key="dataset_select",
-        index=12,  # AG_NEWS
-        help="Select the dataset to work on.",
+        "Dataset", dataset_list, key="dataset_select", index=12, help="Select the dataset to work on.",  # AG_NEWS
```

Comment:
Did these changes come from an automatic syntax formatter? It may be best to revert them if they don't match the style used for the codebase. It also looks like the #AG_NEWS comment got deassociated from its corresponding argument.

Reply:
Sorry, it was by mistake, I reverted the changes.